index.html

```diff
@@ -13,13 +13,27 @@
     <header>
         <nav class="navbar">
             <a class='logo'>Muzaffer Dler</a>
-            <img class="ulnavbar" src="./img/Union.svg" alt="Menu button">
+
+            <div class="desktop-nav">
```

Comment:
- [ ] Kindly align your social icons to the center of the headline section to match the figma design

![image](https://user-images.githubusercontent.com/98453979/193150411-9636f8dc-1552-49f8-88b1-d62feda6717d.png)

### Checkout Figma 

![image](https://user-images.githubusercontent.com/98453979/193150533-304cc3e2-b601-4a6a-93f1-9a0c4b6bfbe9.png)

- [ ] Kindly fix all your buttons border thickness to match the figma design. You can use `border:1px solid color`

![image](https://user-images.githubusercontent.com/98453979/193150877-bf93f6f3-8f7d-4894-8d49-febca77cc951.png)

![image](https://user-images.githubusercontent.com/98453979/193151386-26d847eb-fc2b-40ac-b4c9-a61033bf9a0c.png)


### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/193150994-57d3db41-d1b8-4b73-ad18-f8b4ca3c4127.png)

- [ ] On the Figma design, you will notice that only the first card has `see project` button but all your cards have it. Kindly correct that. Also use different background images like in the Figma design.

![image](https://user-images.githubusercontent.com/98453979/193152043-d76bedb4-ba17-4683-809e-b18e50f9696f.png)

### Checkout  figma

![image](https://user-images.githubusercontent.com/98453979/193152126-f3fa245e-2795-4fa5-be1c-4ffb87d7f4c7.png)

- [ ] Kindly try to align your language and skills section items to the center. They are currently aligned to the right.

![image](https://user-images.githubusercontent.com/98453979/193152398-5ba92c5d-d124-4803-9ac3-4b8320499eef.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/193152518-a2af0118-76c6-429e-832c-4150628e30e2.png)

Reply:
hi thanks for all of that great comment @frank1738 
the see project was on the mobile phone that's why I implemented on the desktop version if you would please check the mobile again and you will see for each project its the sample picture  and for the language and skills section:
![image](https://user-images.githubusercontent.com/83097009/193155973-dcb24800-63b4-44e0-9eaa-922f9fd32333.png)
it looks fine on my end